app/views/pages/home/_collecting_signatures_petitions.html.erb

```diff
@@ -0,0 +1,14 @@
+<% if actioned[:open][:count].zero? %>
+  <p><%= t(:"ui.home_page.open_petitions.none_open") %></p>
+<% else %>
+  <p><%= t(:"ui.home_page.open_petitions.description") %></p>
+  <ol class="threshold-petitions">
+    <% actioned[:open][:list].each do |petition| %>
+      <li class="petition-item">
+        <h3><%= link_to petition.action, petition_path(petition), class: "threshold-petition-title" %></h3>
+        <p class="intro"><%= t(:"ui.home_page.open_petitions.open_date", date: short_date_format(petition.closed_at)) %></p>
+      </li>
+    <% end -%>
+  </ol>
+  <p><%= link_to petition_count(:open_explanation, actioned[:open][:count]), petitions_path(state: :open), class: "view-all" %></p>
+<% end %>
```

Comment:
Is this just not `_open_petitions.html.erb` ? If so might as well just leave it at that.

Reply:
The template is dynamically looked up based on the facet (https://github.com/scottishparliament/e-petitions/blob/fix/petition-states-labelling/app/views/petitions/search/_results.html.erb#L8) 

Do you want me to remove the open one instead?